slurm/2_ApplicationSpecific/VASP-GPU-podman/README.md

```diff
@@ -6,22 +6,22 @@ This is how to build VASP for GPU use, with a sample Slurm script
 
 To build VASP with nvidia GPU support takes about 45 minutes on a debug node.
 
-Get an interactive job on a debug node.
+Start an interactive job on a debug node.
 This example allcates a GPU to the job, but all tests are curenlty commented
 due to issues with using nvidia GPUs with "podman build"
 
 ```
 salloc --partition=debug --qos=debug --nodes=1 --gpus-per-node=1 --cpus-per-task=1 --tasks-per-node=32 --exclusive
 ```
 
-Create a directory for the podman image tarballs e.g.
+Create a directory for the podman OCI images e.g.
 ```
-mkdir -p /projects/academic/group/podman/images
+mkdir -p /projects/academic/ccrgroup/oci_archive_dir
```

Comment:
Can you please update this to use the format that we used throughout our documentation when referring to project directories? We rehashed many options and this may not be the best one but at least it's consistent with what we use everywhere else:  `[YourGroupName]` so `/projects/academic/[YourGroupName]/`

Reply:
I followed what I have in the storage KeyNote/PowerPoint where explanation text
uses [YourGroupName] and [CCRusername]  but terminal examples use (e.g.)
 /ccrgroup/  and  /ccruser/
I felt these fell into the latter category, but I can change if you wish?

Reply:
Yes please.  It can be confusing to users when the formatting is different.  We had hoped that users would understand that square brackets mean "replace me" whereas without them they will often copy/paste the text, not realizing modification was required.

Reply:
done